src/confd/src/ietf-interfaces.c

```diff
@@ -26,7 +26,7 @@
 #define ONOFF(boolean) boolean ? "on" : "off"
 
 #define IF_XPATH "/ietf-interfaces:interfaces/interface"
-
+#define IX_VPATH "%s/infix-interfaces:vlan"
```

Comment:
Both of these are XPATHs, we don't have any VPATHs. So maybe this should be `IX_VLAN_XPATH` or something?

Reply:
Agreed, fixing!